### DISCUSSION TITLE
ESP32 with ethernet

### DISCUSSION THREAD

**Discussion by RAPHCVR** [awawa-dev/HyperSerialESP32#2] [Q&A]
Hello, is the esp ethernet, like the esp wt32-eth01 compatible with this firmware?
If so, is the ethernet connection enabled by default?

**Comment by awawa-dev** [comment]
Hi
esp wt32-eth01 doesn't even have USB port that is used to communicate with host via serial port. I've got one of this device and is possible to make AWA protocol over Ethernet but it's overkill since TCP/UDP already have error detection. And it would require to create a new project since HyperSerial would not fit here ;)

**Comment by appleimperio** [comment]
I'm using it with WLED since the pi is connected to ethernet too the response is very fast! but also is very tricky to load the firmware.
